lib/vanagon/component/dsl.rb

```diff
@@ -244,7 +244,10 @@ def install_configfile(source, target)
       # @param target [String] path to the desired symlink
       def link(source, target)
         @component.install << "#{@component.platform.install} -d '#{File.dirname(target)}'"
-        @component.install << "ln -s '#{source}' '#{target}'"
+        # Use a bash conditional to only create the link if it doesn't already point to the correct source.
+        # This allows rerunning the install step to be idempotent, rather than failing because the link
+        # already exists.
+        @component.install << "([[ '#{target}' -ef '#{source}' ]] || ln -s '#{source}' '#{target}')"
```

Comment:
Oh, I guess some of the concern was if it was a directory. From a `make` perspective we just want to make it a new thing.